lib/Runtime/Language/SourceTextModuleRecord.cpp

```diff
@@ -609,6 +645,15 @@ namespace Js
         }
     }
 
+    void SourceTextModuleRecord::EnsureChildModuleSet(ScriptContext *scriptContext)
+    {
+        if (nullptr == this->childrenModuleSet)
+        {
+            ArenaAllocator* allocator = scriptContext->GeneralAllocator();
+            this->childrenModuleSet = (ChildModuleRecordSet*)AllocatorNew(ArenaAllocator, allocator, ChildModuleRecordSet, allocator);
```

Comment:
Updated in 2nd iteration.

---
In reply to: [114930171](https://github.com/Microsoft/ChakraCore/pull/2913#discussion_r114930171) [](ancestors = 114930171)